### PR TITLE
Fix #58: Resolved CPPLint warnings

### LINKED ISSUE #58: Resolve CPPLint warnings

### PR DESCRIPTION
**Pull request by tmancey**

### CODE REVIEW

**Comment by tmancey** [comment]
Fixes https://github.com/brave-intl/bat-native-ads/issues/58

File: include/bat/ads/ad_info.h [hunk 1]
```diff
@@ -2,7 +2,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#pragma once
+#ifndef SRC_BRAVE_VENDOR_BAT_NATIVE_ADS_INCLUDE_BAT_ADS_AD_INFO_H_
```
Comment (bridiver) [inline include/bat/ads/ad_info.h]: these shouldn't have `SRC_BRAVE_VENDOR_` at the start. If you are getting that full path from lint then something is misconfigured with it

File: src/catalog_state.h [hunk 2]
```diff
@@ -2,7 +2,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#pragma once
+#ifndef SRC_BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_CATALOG_STATE_H_
```
Comment (bridiver) [inline src/catalog_state.h]: this should actually just be `CATALOG_STATE_H_` unfortunately. We should create a directory hierarchy for these to fix that

Comment (tmancey) [reply src/catalog_state.h]: 👍 I will fix by removing SRC_BRAVE_VENDOR_ right now and then can create another ticket once ready to change the directory hierarchy if you are happy with that?

File: include/bat/ads/ad_info.h (hunk 1 above)
Comment (tmancey) [reply include/bat/ads/ad_info.h]: 👍 yes I was getting from lint, so will fix now

Comment (tmancey) [reply include/bat/ads/ad_info.h]: Fixed

File: src/catalog_state.h (hunk 2 above)
Comment (tmancey) [reply src/catalog_state.h]: Fixed

File: include/bat/ads/ad_info.h [hunk 3]
```diff
@@ -2,7 +2,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#pragma once
+#ifndef BAT_NATIVE_ADS_INCLUDE_BAT_ADS_AD_INFO_H_
```
Comment (bridiver) [inline include/bat/ads/ad_info.h]: sorry, I missed that there was also some additional incorrect path. It should always match the include path which is BAT_ADS_AD_INFO_H_

Comment (tmancey) [reply include/bat/ads/ad_info.h]: Fixed

**Pull request merged**